hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/ControllerManagementTest.java

```diff
@@ -515,6 +515,21 @@ public void findOrRegisterTargetIfItDoesNotExist() {
         assertThat(targetRepository.count()).as("Only 1 target should be registred").isEqualTo(1L);
     }
 
+    @Test
+    @Description("Register a controller with name which does not exist")
+    @ExpectEvents({ @Expect(type = TargetCreatedEvent.class, count = 1),
+            @Expect(type = TargetPollEvent.class, count = 2) })
+    public void findOrRegisterTargetIfItDoesNotExistWithName() {
```

Comment:
instead of duplicating the logic of `findOrRegisterTargetIfItDoesNotExist ` check if the name is really changed for the same target here

Reply:
Done.